core/platform/models_test.py

```diff
@@ -409,4 +404,4 @@ def test_import_models_not_implemented_has_not_implemented_error(
             re.escape(
                 'import_models() method is not overwritten in '
                 'derived classes')):
-            models.Platform().import_models([models.NAMES.base_model])
+            models.Platform().import_models([models.NAMES.base_model.value])
```

Comment:
Why is this changed here but not in other places?

Reply:
Done